mypy.ini

```diff
@@ -0,0 +1,25 @@
+[mypy]
+allow_redefinition = False
+allow_untyped_globals = False
+check_untyped_defs = True
+disallow_incomplete_defs = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_decorators = True
+disallow_untyped_defs = True
+follow_imports = silent
+ignore_missing_imports = False
+implicit_reexport = False
+no_implicit_optional = True
+pretty = True
+show_error_codes = True
+strict_equality = True
+warn_no_return = True
+warn_return_any = True
+warn_unreachable = True
+warn_unused_ignores = True
+
+# Bundled third-party package.
```

Comment:
Looks like something before 0.4.1. If we use current docopt and the `types-docopt` package, we would get its annotation. I don't know any reason for bundling it now.

Reply:
well.. me neither. Maybe it was to limit the needed dependencies to bare none by then 🤔
I suppose I could use click for that, but we can migrate to the dependency I suppose. The codebase wasn't refreshed  since 2012 (except for black) and future python versions might require unintended changes that will have to be tested (and it's not something I'd like to do)